pkg/proto/query_coord.proto

```diff
@@ -995,6 +997,38 @@ message RunAnalyzerRequest{
     bool with_hash = 7;
 }
 
+message HighlightTask{
+    string field_name = 1;
+    int64 field_id = 2;
+    repeated string target_texts = 3;
+    repeated string target_analyzers = 4;
+
+    repeated string texts = 5;
+    repeated string analyzers = 6;
+}
+
+message GetHighlightRequest{
+    common.MsgBase base = 1;
+    string channel = 2;
+    repeated int64 topks = 3;
+    repeated HighlightTask tasks=4;
+}
+
+message HighlightFragment{      
+    int64 start_offset = 1;
+    int64 end_offset = 2;
+    repeated int64 offsets = 3;
+}
+
+message HighlightResult{       
+    repeated HighlightFragment fragments = 2;
+}
+
+message GetHighlightResponse{
```

Comment:
in the case of nq>1, 1 GetHighlightResponse is for 1 query?

add comment:

Each query has 1 GetHighlightResponse, with 1 HighlightResult for each of top k; each HighlightResult has multuple fragments(defined by start_offset to end_offset), each fragments may have multiple highlighted words(defined by offsets, whose size must be a multiple of 2).